index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  
+  <head>
+    <title>Startup Matchmaker</title>
+    <link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700|Oswald:700|Merriweather:700,700italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+  </head>
+
+
+  <body>
+
+    <header>
+
+    <div class="nav">
+      <h1>Startup Matchmaker</h1>
+      <nav>
+        <h3><a href="#">Find Developers</a></h3>
```

Comment:
@leebovel Since these are links in a navigation I would just use an `</li>` instead of having `</h3>` &  `</h4>`.  Heading tags are used more for content heading to organize the content in the document 🔨